### PR TITLE
add mrsky and update some other things

### PR DESCRIPTION
**Pull request by angle943**

### CODE REVIEW

**Comment by angle943** [comment]
1) cleaned up the logic. We had it where a user acn be involved with multiple teams, and thus have multiple titles. We got rid of this rule, so I cleaned it up to be singular values for both.

2) added the ability to filter the team member page via the position, using queryParams. I did this because from the `Projects` page, I want to be able to link to just the project reviewers

File: src/containers/Teams/index.tsx [hunk 1]
```diff
@@ -87,26 +91,18 @@ const Teams: FC = () => {
...
-            teamLeads.push({...member, isLead: true});
-          } else {
-            otherMembers.push({...member, isLead: false});
-          }
-        }
-      });
-      return teamLeads.concat(otherMembers);
-    };
+    const getFilteredMembers = () =>
+      teamMembers.filter((teamMember) => teamMember.team.toLowerCase() === teamFilter.toLowerCase());
```
Comment (jamessspanggg) [inline src/containers/Teams/index.tsx:95]: is the order of `lead` and normal team members not matter anymore?

File: src/containers/Teams/index.tsx [hunk 2]
```diff
@@ -87,26 +91,18 @@ const Teams: FC = () => {
...
+
     const members = teamFilter === TeamName.all ? teamMembers : getFilteredMembers();
     const sortedMembers = sortTeamMembers(members);
-    setFilteredMembers(sortedMembers);
-  }, [teamFilter]);
+
+    if (queryTitle) {
+      setFilteredMembers(sortedMembers.filter((member) => member.title.toLowerCase() === queryTitle.toLowerCase()));
```
Comment (jamessspanggg) [inline src/containers/Teams/index.tsx:101]: what is this change for? how can we set the `queryTitle` within our website? These changes are causing quite a few merge conflicts to https://github.com/thenewboston-developers/Website/pull/1663 but not sure to follow this implementation

Comment (jamessspanggg) [reply src/containers/Teams/index.tsx:101]: what's the use case for this? it seems like only if we add `title` query param manually in the url, then we will filter the teams based on member

Comment (jamessspanggg) [reply src/containers/Teams/index.tsx:101]: ok saw ur explanation above, ignore this

File: src/containers/Teams/index.tsx (hunk 1 above)
Comment (angle943) [reply src/containers/Teams/index.tsx:95]: what do you mean?

Comment (thesanjeevsharma) [reply src/containers/Teams/index.tsx:95]: @jamessspanggg On line 98, there's `sortTeamMembers` function. That takes care of it.

**Pull request merged**